app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -15,25 +15,28 @@ import kotlinx.serialization.json.jsonArray
 import kotlinx.serialization.json.jsonObject
 import kotlinx.serialization.json.jsonPrimitive
 import java.net.URL
+import java.security.MessageDigest
 
 // This is an object - a static object if you will that will exist in the global context. There
 // will always be one and only one instance of this object
 object GameState {
     // Class fields
     var gameID = ""
-    var players = mutableListOf<Player>()
-        private set
+    val players = mutableListOf<Player>()
     var startingFunds: Int = -1
     var smallBlind: Int = -1
+    var isPlayerAdmin: Boolean = false
 
     // Callbacks
-    private var playerJoinedCallbacks = HashMap<Int, (Player) -> Unit>()
-    private var playerRemovedCallbacks = HashMap<Int, (Player) -> Unit>()
+    var onGameReset = {}
+    private val playerJoinedCallbacks = HashMap<Int, (Player) -> Unit>()
+    private val playerRemovedCallbacks = HashMap<Int, (Player) -> Unit>()
+    private var SettingsChangedCallback = HashMap<Int, () -> Unit>()
```

Comment:
Please use lower camel case for this `settingsChangedCallback`

Reply:
Done.